semantic_conventions/metrics/faas.yaml

```
@@ -0,0 +1,20 @@
+groups:
+    - id: faas-metrics
+      prefix: faas
+      brief: >
+          This document defines the attributes used in
+          faas (function as a service) metrics.
+      attributes:
+        - ref: faas.trigger
+          required: always
+        - ref: faas.invoked_name
+          required: always
+        - ref: faas.invoked_provider
+          required: always
+        - ref: faas.invoked_region
+          required: always
+        - ref: faas.coldstart
+          required: always
+        - id: faas.error
```

```suggestion
        - id: error
```
Otherwise, the tool will understand this as `faas.faas.error`. The value of `prefix` is always prepended to new attributes whereas ref attributes require the full identifier :sweat_smile: